.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           toolchain: stable
           components: rustfmt
 
-      - run: cargo fmt -- --check
+      - run: rustfmt --edition 2021 **/*.rs --check
```

Comment:
Why use `rustfmt` directly instead of `cargo fmt`?

Reply:
Hey I copied it in from a different repo but ya we can use either here.